src/main/java/mate.academy/service/ProductParser.java

```diff
@@ -1,7 +1,9 @@
 package mate.academy.service;
 
+import mate.academy.lib.Component;
 import mate.academy.model.Product;
 
+@Component
 public interface ProductParser {
```

Comment:
Convention is to use Component on the classes that are used for instance creation (usually these are interface implementations)